src/sentry/integrations/utils/atlassian_connect.py

```diff
@@ -88,7 +90,7 @@ def get_integration_from_jwt(
 
 
 def verify_claims(
-    claims: Optional[Mapping[str, str]],
+    claims: Mapping[str, str],
```

Comment:
How did you noticed that this does not have to be optional anymore?

Reply:
I traced the output of the `jwt` methods and noticed they only raise or return a dict